conf/proxy.conf

```diff
@@ -253,3 +256,9 @@ clusterName=
 
 # Deprecated. Use configurationStoreServers
 globalZookeeperServers=
+
+# The ZooKeeper quorum connection string (as a comma-separated list)
```

Comment:
Add `### --- Deprecated settings --- ###` before?

Reply:
It's under ### --- Deprecated config variables --- ###